public/js/project.js

```diff
@@ -86,6 +87,16 @@ $('#new-project').on('show.bs.modal', function() {
     webBtn.addClass('btn-primary');
   });
 
+  $("#empty-checkbox").on('change', function() {
+    emptyProject = this.checked;
+    if (emptyProject) {
+      templateBtn.removeClass('btn-primary');
+      sampleBtn.removeClass('btn-primary');
+    } else {
+      templateBtn.addClass('btn-primary');
+    }
```

Comment:
What about sampleBtn?

Reply:
sampleBtn? We deselect the templateBtn and the sampleBtn when clicking the empty checkbox.
On the contrary, If we deselect the empty checkbox then activating templateBtn.

Reply:
In case of 'emptyProject == false', should only templateBtn be activated ?

Reply:
Yes. That is default.

Reply:
OK.